demangle/vendor/swift/README.md

```diff
@@ -0,0 +1,61 @@
+# Vendored Swift Demangler
+
+This folder contains a vendored subset of the [Swift Programming Language]. The Swift library is
+reduced to the demangler only to reduce the size of this package.
+
+The current version is **Swift 5.2.1**.
+
+## Sentry Modifications
+
+The library has been modified to add an option to hide function arguments during demangling. This
+patch is maintained in `1-arguments.patch`.
+
+## How to Update
+
+1. **Check out the [latest release] of Swift:**
+   1. Create a directory that will house swift and its dependencies:
+      ```
+      $ mkdir swift-source && cd swift-source
+      ```
+   2. Clone the swift repository into a subdirectory:
+      ```
+      $ git clone https://github.com/apple/swift.git
+      ```
+   3. Check out dependencies:
+      ```
+      $ ./swift/utils/update-checkout --clone-with-ssh
+      ```
+   4. Check out the release banch of the latest release:
+      ```
+      $ git checkout swift-5.2.1-RELEASE
+      ```
+   5. Build the complete swift project (be very patient, this may take long):
+      ```
+      $ cd swift
+      $ ./utils/build-script
```

Comment:
I think you can use `--skip-build` here. The headers in `build/Ninja-DebugAssert/*-macosx-x86_64/include` are still generated when `--skip-build` is used. This avoids building all of swift.